File: examples/python/chat_completion.py
#!/usr/bin/env python3

import os
from openai import OpenAI

client = OpenAI(base_url="http://127.0.0.1:3030/v1")

completion = client.chat.completions.create(
    messages=[
        {
            "role": "user",
            "content": "Say this is a test",
        }
    ],
    model="gpt-4o-mini",
)

print(completion.choices[0].message.content)

File: examples/node/chat_completion.js
// node ./chat_completion.js 
import OpenAI from "openai";

const openai = new OpenAI({
	baseURL: 'http://127.0.0.1:3030/v1'
});

async function main() {
  const completion = await openai.chat.completions.create({
    model: 'gpt-4o-mini',
    messages: [{ role: 'user', content: 'Say this is a test' }],
  });
  console.log(completion.choices[0]?.message?.content);
}

main();
